packages/menu-bar/src/vaadin-menu-bar.js

```diff
@@ -200,6 +201,30 @@ class MenuBar extends ButtonsMixin(InteractionsMixin(ElementMixin(ThemableMixin(
     };
   }
 
+  /**
+   * Override method inherited from `DisabledMixin`
+   * to update the `disabled` property for the buttons
+   * whenever the property changes on the menu bar.
+   *
+   * @param {boolean} newValue the new disabled value
+   * @param {boolean} oldValue the previous disabled value
+   * @override
+   * @protected
+   */
+  _disabledChanged(newValue, oldValue) {
+    super._disabledChanged(newValue, oldValue);
+    if (oldValue !== newValue) {
+      this.__updateItemsDisabledProperty();
```

Comment:
Let's rename the method  to `__updateButtonsDisabled()` as it updates buttons, not the items.

Reply:
I was originally changing the item's disabled property but then had to change to the buttons. I forgot to rename the method accordingly. Thanks.